example.config.yml

```diff
@@ -69,6 +69,19 @@ apache_vhosts:
   - {servername: "xhprof.drupalvm.dev", documentroot: "/usr/share/php/xhprof_html"}
   - {servername: "pimpmylog.drupalvm.dev", documentroot: "/usr/share/php/pimpmylog"}
 
+# Enable SSL Support on the Drupal site vhosts.
+# See https://github.com/geerlingguy/ansible-role-apache for additional options.
+# Update the file paths below if you are using a custom SSL certificate.
+apache_vhosts_ssl:
+  - {
+    servername: "{{ drupal_domain }}",
+#    serveralias: "www.{{ drupal_domain }}",
+    documentroot: "{{ drupal_core_path }}",
+    certificate_file: "/etc/ssl/certs/ssl-cert-snakeoil.pem",
```

Comment:
I can definitely do a little rewriting of the comments.  We could also just add either a separate example config and/or a wiki page on SSL support as short term fixes.

Since it's almost a given that in the near future all sites are going to need to be SSL encrypted, I think it would make sense to be able to have the drupal-vm support SSL out of the box.  Is there a good Ansible SSL library that you recommend?

What if we modified the provisioning script so that we generate a self-signed SSL certificate for "*.{{ drupal_domain }}" as part of the provisioning and reference that certificate in the vhost config?